server/src/format.ts

```diff
@@ -11,11 +11,11 @@ export default async function format(
   } catch (errors) {
     if (onError) {
       onError(
-        "prisma-fmt error'd during formatting. This was likely due to a syntax error. To get a more detailed output please see Prisma Language Server output. To see the output, go to View > Output from the toolbar, then select 'Prisma Language Server' in the Output panel.",
+        "prisma-fmt error'd during formatting. To get a more detailed output please see Prisma Language Server output. To see the output, go to View > Output from the toolbar, then select 'Prisma Language Server' in the Output panel.",
       )
     }
     console.warn(
-      "\nprisma-fmt error'd during formatting. This was likely due to a syntax error. If not, please report this issue on [Prisma VSCode](https://github.com/prisma/vscode/issues). \nLinter output:\n",
+      "\nprisma-fmt error'd during formatting. Please report this issue on [Prisma VSCode](https://github.com/prisma/vscode/issues). \nLinter output:\n",
     )
```

Comment:
Unrelated? Different base branch?

Reply:
Yes, this is unrelated, but since the new prisma-fmt does not throw any errors anymore on wrong formatting, this will not likely be due to a syntax error anymore. I could create an issue and a separate PR for this though?

Reply:
Not that important - was just concerned this might slip in undetected.